bindings/nodejs/lib/types/models/info/node-info-protocol.ts

```diff
@@ -102,6 +102,46 @@ export interface ProtocolParameters {
      * The parameters used by signaling protocol parameters upgrade.
      */
     versionSignaling: VersionSignalingParameters;
+    /**
+     * Rewards Parameters defines the parameters that are used to calculate Mana rewards.
+     */
+    rewardsParameters: RewardsParameters;
+}
+
+/**
+ * Rewards Parameters defines the parameters that are used to calculate Mana rewards.
+ */
+export interface RewardsParameters {
+    /**
+     * The number of validation blocks that should be issued by a selected validator
+     * per slot during its epoch duties.
+     */
+    validationBlocksPerSlot: number;
```

Comment:
This is `validator_blocks_per_slot` in Rust so one of them is wrong

Reply:
Has Python been done already? If yes, double check?

Reply:
https://github.com/iotaledger/tips-draft/pull/25#issuecomment-1755703335

Reply:
Lets make it validator_blocks_per_slot and change when they comment. no point keeping stuff open

Reply:
This is ok now.